tests/src/tracing/tracecontrol_api/Configuration.cs

```diff
@@ -0,0 +1,110 @@
+using System;
```

Comment:
Can you also change the namespace in the code from EventPipe to Tracing.Tests.Common?